EIPS/eip-0000.md

```diff
@@ -0,0 +1,152 @@
+---
+eip: 0000
+title: Multidimensional Gas Metering
+description: Gas accounting by EVM resource, increasing throughput and improving resource usage controls, with minimal changes to the protocol and UX
+author: Maria Silva (@misilva73), Davide Crapis (@dcrapis), Anders Elowsson (@anderselowsson), Toni Wahrstätter (@nerolation)
+discussions-to: 
```

Comment:
Please add a discussions topic in Eth Magicians using the template.
https://ethereum-magicians.org/c/eips/5